translations/monero-core_fr.ts

```diff
@@ -150,7 +150,7 @@
     <message>
         <location filename="../pages/History.qml" line="70"/>
         <source> selected: </source>
-        <translation> séléctionné : </translation>
+        <translation> sélectionné&nbsp;: </translation>
```

Comment:
French typography requires non-breaking spaces (and in some instance, thin non-break spaces)